.env

```diff
@@ -0,0 +1,2 @@
+GOVUK_NOTIFY_API_KEY=EXAMPLE_API_KEY
+GOVUK_NOTIFY_TEMPLATE_ID_REGISTRATION=EXAMPLE_TEMPLATE_ID_REGISTRATION
```

Comment:
Great that these are kept separate and we'll use CDP secrets for them for deployed code.

For local, I think we can follow the pattern set out [here](https://github.com/DEFRA/epr-backend/blob/main/compose/aws.env) and [here](https://github.com/DEFRA/epr-backend/blob/main/compose/start-localstack.sh), although the duplication feels a bit messy. Happy to chat this through to find a suitable solution, but I don't think we need to introduce dotenv